File: Exercise 4/Java/src/main/java/de/thu/recipebook/models/Recipe.java
package de.thu.recipebook.models;

public class Recipe {
    private String name;
    private String country;
    private String ingredients;
    private String instructions;

    public Recipe(String name, String country, String ingredients, String instructions) {
        this.name = name;
        this.country = country;
        this.ingredients = ingredients;
        this.instructions = instructions;
    }

    public String getName() {
        return name;
    }

    public String getCountry() {
        return country;
    }

    public String getIngredients() {
        return ingredients;
    }

    public String getInstructions() {
        return instructions;
    }

    @Override
    public String toString() {
        return name + "\n" + "Country of origin:\n" + country + "\nIngredients:\n" + ingredients
                + "\nInstructions:\n" + instructions;
    }
}

File: Exercise 4/Kotlin/src/main/java/de/thu/recipebook/models/Recipe.kt
package de.thu.recipebook.models

class Recipe(
    val name: String,
    val country: String,
    val ingredients: String,
    val instructions: String
) {

    override fun toString(): String {
        return "$name\nCountry of origin:\n$country\nIngredients:\n$ingredients\nInstructions:\n$instructions"
    }
}